packages/apps/human-app/server/src/common/config/environment-config.service.ts

```diff
@@ -14,10 +14,18 @@ export class EnvironmentConfigService {
   get reputationOracleUrl(): string {
     return this.configService.get<string>('REPUTATION_ORACLE_URL', '');
   }
+  get e2eTestingEmailAddress(): string {
+    return this.configService.get<string>('E2E_TESTING_EMAIL_ADDRESS', '');
+  }
+  get e2eTestingPassword(): string {
+    return this.configService.get<string>('E2E_TESTING_PASSWORD', '');
+  }
```

Comment:
test related stuff shouldn't be in the production code. It should be in test directory or at least a separate file for test config

Reply:
Solved in 4865ae76